README.rst

```diff
@@ -13,1837 +13,30 @@ Turn complex GraphQL queries into optimized database queries.
-vertex :code:`b` also be connected to both :code:`x` and :code:`y` via directed
-edges of type :code:`E`.
-
-Consider the GraphQL query:
-
-.. code::
-
-    {
-        S {
-            name @output(out_name: "s_name")
-            out_E {
-                name @output(out_name: "t_name")
-            }
-        }
-    }
-
-Between the data in the database and the query's structure, it is clear
-that combining any of :code:`a` or :code:`b` with any of :code:`x` or :code:`y` would
-produce a valid result. Therefore, the complete result list, shown here
-in JSON format, would be:
-
-.. code:: python
-
-    [
-        {"s_name": "a", "t_name": "x"},
-        {"s_name": "a", "t_name": "y"},
-        {"s_name": "b", "t_name": "x"},
-        {"s_name": "b", "t_name": "y"},
-    ]
-
-This is precisely what the :code:`MATCH` compilation target is guaranteed to
-produce. The remainder of this section is only applicable to the
-:code:`gremlin` compilation target. If using :code:`MATCH`, all of the queries
-listed in the remainder of this section will produce the same, complete
-result list.
-
-Since the :code:`gremlin` compilation target does not guarantee a complete
-result list, querying the database using a query string generated by the
-:code:`gremlin` compilation target will produce only a partial result list
-resembling the following:
-
-.. code:: python
-
-    [
-        {"s_name": "a", "t_name": "x"},
-        {"s_name": "b", "t_name": "x"},
-    ]
-
-Due to limitations in the underlying query language, :code:`gremlin` will by
-default produce at most one result for each of the starting locations in
-the query. The above GraphQL query started at the type :code:`S`, so each
-:code:`s_name` in the returned result list is therefore distinct.
-Furthermore, there is no guarantee (and no way to know ahead of time)
-whether :code:`x` or :code:`y` will be returned as the :code:`t_name` value in each
-result, as they are both valid results.
-
-Users may apply the :code:`@output_source` directive on the last scope of
-the query to alter this behavior:
-
-.. code::
-
-    {
-        S {
-            name @output(out_name: "s_name")
-            out_E @output_source {
-                name @output(out_name: "t_name")
-            }
-        }
-    }
-
-Rather than producing at most one result for each :code:`S`, the query will
-now produce at most one result for each distinct value that can be found
-at :code:`out_E`, where the directive is applied:
-
-.. code:: python
-
-    [
-        {"s_name": "a", "t_name": "x"},
-        {"s_name": "a", "t_name": "y"},
-    ]
-
-Conceptually, applying the :code:`@output_source` directive makes it as if
-the query were written in the opposite order:
-
-.. code::
-
-    {
-        T {
-            name @output(out_name: "t_name")
-            in_E {
-                name @output(out_name: "s_name")
-            }
-        }
-    }
-
-SQL
----
```

Comment:
The SQL section is essentially copy pasted in ReadTheDocs. Verified this by using git diff.